extensions/websockets-next/server/runtime/src/main/java/io/quarkus/websockets/next/WebSocketRuntimeConfig.java

```diff
@@ -0,0 +1,21 @@
+package io.quarkus.websockets.next;
+
+import java.time.Duration;
+import java.util.Optional;
+
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+import io.smallrye.config.ConfigMapping;
+
+@ConfigMapping(prefix = "quarkus.web-socket-next")
```

Comment:
Should the prefix perhaps be `quarkus.websocket-next`? I know the spelling `WebSocket` suggests these are two words, but `quarkus.websocket-next` seems more ergonomic than `quarkus.web-socket-next`.

Also, the extension is called `websockets-next`. We should decide whether it's singular or plural and then stick to it :-)

Reply:
Hm, the `quarkus.websockets-next` makes more sense. OTOH the config is currently not used at all. So it's up for discussion.